### ISSUE TITLE
RuntimeError: Failed easy perform [code: 7] Couldn't connect to server

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/openvstorage-health-check#97]
```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/celery/local.py", line 188, in __call__
    return self._get_current_object()(*a, **kw)
  File "/usr/local/lib/python2.7/dist-packages/celery/app/task.py", line 420, in __call__
    return self.run(*args, **kwargs)
  File "ovs/lib/healthcheck.py", line 86, in check_attended
    return HealthCheckController.execute_check()
  File "/usr/local/lib/python2.7/dist-packages/celery/local.py", line 188, in __call__
    return self._get_current_object()(*a, **kw)
  File "/usr/local/lib/python2.7/dist-packages/celery/app/task.py", line 420, in __call__
    return self.run(*args, **kwargs)
  File "ovs/lib/healthcheck.py", line 127, in execute_check
    HealthCheckController.check_openvstorage()
  File "/usr/local/lib/python2.7/dist-packages/celery/local.py", line 188, in __call__
    return self._get_current_object()(*a, **kw)
  File "/usr/local/lib/python2.7/dist-packages/celery/app/task.py", line 420, in __call__
    return self.run(*args, **kwargs)
  File "ovs/lib/healthcheck.py", line 174, in check_openvstorage
    ovs.check_model_consistency()
  File "ovs/extensions/healthcheck/openvstorage/openvstoragecluster_health_check.py", line 826, in check_model_consistency
    voldrv_client = src.LocalStorageRouterClient(config_file)
RuntimeError: Failed easy perform [code: 7] Couldn't connect to server

```

**Comment by wimpers** [comment]
Some context might be handy.

**Comment by kinvaris** [comment]
@wimpers well this error should just be caught and displayed as: `volumedriver is probably offline` or something.

**Comment by kinvaris** [comment]
FAILED: As seen on a gig environment
```
[INFO] Checking vPool 'data01': 
Traceback (most recent call last):
File "/opt/OpenvStorage/ovs/lib/healthcheck.py", line 35, in <module>
class HealthCheckController(object):
File "/opt/OpenvStorage/ovs/lib/healthcheck.py", line 414, in HealthCheckController
HealthCheckController.run_method(*arguments)
File "/opt/OpenvStorage/ovs/lib/healthcheck.py", line 363, in run_method
return HealthCheckController.check_attended()
File "/usr/lib/python2.7/dist-packages/celery/local.py", line 188, in _call_
return self._get_current_object()(*a, **kw)
File "/usr/lib/python2.7/dist-packages/celery/app/task.py", line 420, in _call_
return self.run(*args, **kwargs)
File "/opt/OpenvStorage/ovs/lib/healthcheck.py", line 70, in check_attended
return HealthCheckController.execute_check(unattended, silent_mode)
File "/usr/lib/python2.7/dist-packages/celery/local.py", line 188, in _call_
return self._get_current_object()(*a, **kw)
File "/usr/lib/python2.7/dist-packages/celery/app/task.py", line 420, in _call_
return self.run(*args, **kwargs)
File "/opt/OpenvStorage/ovs/lib/healthcheck.py", line 107, in execute_check
HealthCheckController.check_openvstorage(logger)
File "/usr/lib/python2.7/dist-packages/celery/local.py", line 188, in _call_
return self._get_current_object()(*a, **kw)
File "/usr/lib/python2.7/dist-packages/celery/app/task.py", line 420, in _call_
return self.run(*args, **kwargs)
File "/opt/OpenvStorage/ovs/lib/healthcheck.py", line 131, in check_openvstorage
OpenvStorageHealthCheck.run(logger)
File "/opt/OpenvStorage/ovs/extensions/healthcheck/openvstorage/openvstoragecluster_health_check.py", line 754, in run
OpenvStorageHealthCheck.check_for_halted_volumes(logger)
File "/opt/OpenvStorage/ovs/extensions/healthcheck/openvstorage/openvstoragecluster_health_check.py", line 698, in check_for_halted_volumes
if int(OpenvStorageHealthCheck._info_volume(voldrv_client, volume).halted):
File "/usr/local/lib/python2.7/dist-packages/timeout_decorator/timeout_decorator.py", line 69, in new_function
return function(*args, **kwargs)
File "/opt/OpenvStorage/ovs/extensions/healthcheck/openvstorage/openvstoragecluster_health_check.py", line 739, in _info_volume
return voldrv_client.info_volume(volume_name)
volumedriver.storagerouter.storagerouterclient.ClusterNotReachableException:
```

This should also be caught in the same context

**Comment by JeffreyDevloo** [comment]
Fixed by #206
